tests/std/tests/usual_17_matrix.lst

```diff
@@ -24,6 +24,7 @@ PM_CL="/EHsc /MTd /D_ITERATOR_DEBUG_LEVEL=2 /std:c++latest /await"
 PM_CL="/EHsc /MTd /D_ITERATOR_DEBUG_LEVEL=2 /std:c++latest /analyze:only"
 PM_CL="/Za /EHsc /MD /std:c++latest /permissive-"
 PM_CL="/Za /EHsc /MDd /std:c++latest /permissive-"
+PM_CL="/EHsc /MD /std:c++17 /permissive- /Zc:preprocessor /wd5105"

```

Comment:
I still would kind of appreciate if this went at the end of this file.

Reply:
Line 17 seems to correspond to the config that you updated in `usual_17_winsdk_matrix.lst`: both are IDL2 `/std:c++17 /permissive-`.